base/visualization/R/map.output.R

```diff
@@ -15,12 +15,12 @@ map.output <- function(table, variable) {
   map <- ggplot2::ggplot() +
     ggplot2::geom_polygon(
       data = world,
-      ggplot2::aes(x = long, y = lat, group = group),
+      ggplot2::aes(x = world$long, y = world$lat, group = world$group),
```

Comment:
This usage is discouraged by ggplot and will generate a warning message. Instead of the literal dataset name, tell R these variables are [data-masked](https://rlang.r-lib.org/reference/topic-data-mask.html) by prefixing them with the [{rlang} `.data` adverb](https://rlang.r-lib.org/reference/dot-data.html).

Reply:
Thanks. Made the required changes.